src/redux/actions/moviesAction.ts

```diff
@@ -0,0 +1,53 @@
+import { AppDispatch } from "../../types/redux.interface";
+import Movie from "../../types/movie.interface";
+import { GET_MOVIES_ERROR, GET_MOVIES_START, GET_MOVIES_SUCCESS } from "../types";
+
+const BACKEND_URL = "http://localhost:4000/movies?";
+
+export function getAllMoviesAction(numPage: number, sortBy: string) {
+    return async(dispatch: AppDispatch) => {
+        dispatch(getAllMoviesStart());
+
+        try {
+            const response = await fetch(`${ BACKEND_URL }limit=9&offset=${ numPage }&sortBy=${ sortBy }&sortOrder=desc`);
```

Comment:
I guess you can try [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) instead of this string

Reply:
resolved, thank you for the tip